tests/when_resetting_projections.php

```diff
@@ -68,7 +68,7 @@ public function should_reset_the_projection(): void
             );
             $status = $projectionStatus['status'];
 
-            $this->assertStringStartsWith('Preparing', $status);
+            $this->assertTrue(\in_array($status, ['Running', 'Preparing', 'Preparing/Stopping']));
```

Comment:
It's a good practice to add `true` as the third parameter for `in_array` calls.

Reply:
good catch, I missed that